configure.ac

```diff
@@ -302,12 +307,17 @@ if test "$enable_dev_build" = "yes"; then
       AC_C_FLAG([-g3])
       AC_C_FLAG([-O0])
    fi
+
+   if test "$cc_is_gcc" = "yes"; then
+      AC_C_FLAG([-ggdb3])
```

Comment:
clang supports this option too, so the check for gcc is counterproductive.  AFAIK even ICC supports it, not that we really care.  This doesn't need any check really, either the flag works or it doesn't...